staging/script.xsqueeze/resources/lib/classes/NowPlayingWindow.py

```diff
@@ -8,6 +8,13 @@
 import datetime
 import constants
 
+#hdmi support
+if constants.CECSUPPORT == 'true':
+  import cec
+  cec.init()
+  tv   = cec.Device(0)
+  avr  = cec.Device(5)
+
```

Comment:
Hmm, this won't fly for others right?  As in those devices will have different IDs for different people?

I think you'd need to manage this in settings (e.g. see how I do the audio device or squeeze server settings stuff) - I call the addon in a different way and it gives back a list of things to choose.  You'd need to give them a chance to choose the device IDs to control (or at least input a number and give some instructions on how to work out what these numbers should be somewhere).  And how would you deal with e.g. them not using a receiver (or perhaps the pipe the sound back out of their tv via hdmi ARC).